tests/protocols/tfa2017.h

```diff
@@ -0,0 +1,15 @@
+static const int tfa2017_nrtests = 2;
+static const struct raw_t tfa2017_tests[] = {
+	{
+		"4752 1032 912 1039 436 613 841 594 373 1056 431 532 443 535 436 537 927 552 415 558 418 561 416 1037 920 1033 922 555 420 557 416 1034 923 557 424 546 436 1014 453 516 944 535 435 546 444 998 473 506 944 1012 943 1007 475 496 962 526 446 526 446 996 486 490 488 489 962 517 465 516 457 514 466 510 465 985 495 497 490 472 498 476 504 488 491 473 495 475 507 467 509 473 505 465 507 496 961 973 976 966 987 496 488 486 496 946 1012 940 536 438 1027 453 526 921 547 436 540 426 568 408 1051 435 534 445 517 455 530 921 1027 931 1014 465 526 447 528 921 1034 446 526 455 516 931 547 428 1025 461 515 465 514 934 558 414 1045 908 1015 940 540 435 1023 458 509 480 505 931 556 414 567 416 1023 455 528 448 525 456 517 456 517 932 547 436 535 435 557 417 558 425 546 426 546 435 536 441 534 445 539 434 537 445 1004 942 1035 917 1030 459 511 457 517 933 1015 941 537 445 1004 468 517 933 547 435 554 419 543 434 1023 469 500 470 499 476 504 942 1025 924 1030 451 516 466 505 942 1013 466 506 476 496 952 536 435 1017 468 510 468 506 941 528 447 1015 940 1003 959 521 446 1013 465 518 456 517 933 537 446 530 448 1001 477 499 475 509 472 496 476 507 506 297579",
```

Comment:
Maybe add a test with a negative temperature?

Reply:
Will do as soon as it is cold enough, because all my tests are based on real measurements.